stdlib/3.3/lzma.pyi

```diff
@@ -0,0 +1,105 @@
+import io
+import sys
+from typing import Any, BinaryIO, IO, List, Mapping, Optional, Union
+
+if sys.version_info >= (3, 6):
+    from os import PathLike
+    _PathOrFile = Union[str, bytes, IO[Any], PathLike[Any]]
+elif sys.version_info >= (3, 3):
+    _PathOrFile = Union[str, bytes, IO[Any]]
+else:
```

Comment:
Why do you need this case? This module doesn't exist before 3.3.

Reply:
2ae4a6d022c6aebc6a87b9052f7ee4f0d98380b1; I copy/pasted from the bz2 stubs but failed to remove this.